activiti-cloud-starter-query/src/test/java/org/activiti/cloud/starter/tests/QueryTasksIT.java

```diff
@@ -940,4 +940,71 @@ public void shouldGetAvailableStandaloneTasksFilteredByNameDescription() {
         });
         
     }
+    
+    @Test
+    public void shouldCheckTaskProcessDefinitionVersion() {
+      //given
+        CloudTaskCreatedEventImpl e1,e2;
```

Comment:
@CTI777 please use one variable declaration per line and prefer to declare them closer to the usage. I'm updating it.